auto_parking/scripts/parking.py

```diff
@@ -60,8 +61,7 @@ def process_scan(self, m):
                 rospy.signal_shutdown("Done parking.")
             elif self.dist2Neato < 0.3:
                 print "Neato was too close to park!"
-            
-          
+               
```

Comment:
I just realized: should we call rospy.signal_shutdown("Neato is too close to park.") right here?